ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -1,5 +1,8 @@
 <div id="tags" class="tab-section">
   <div role="tabpanel">
+  {{#if topology.FilteredByACLs}}
+    <TopologyMetrics::Banner />
```

Comment:
Should we try and stick to the `<TopologyMetrics::Notice::LimitedAccess />` pattern? I know we've not been great with sticking to patterns due to all the churn lately, but we should start to try to form patterns that we are both happy with asap so we can move forwards with them.

I think here the `Banner` vs `Notice` thing stands out to me (it's using the Notice component) and if you sub folder it, it means if we get any more types of Notices we can put them in the Topology/Notice/Name folder - do you think that sounds ok?